src/main/java/duke/DukeExceptions.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+<<<<<<< HEAD
```

Comment:
I think you accidentally committed this file without resolving conflicts! I noticed this in many of your files.